stats/DataAggregation.md

```diff
@@ -0,0 +1,69 @@
+# Data Aggregation API Overview
+The stats library MUST allow users to define measurements how data are aggregated and broken down.
+The core data types used are:
+* `Aggregation`: defines the aggregation type (e.g. `Distribution`, `Sum`, `Mean`).
+* `View`: defines how individual measurements are broken down and aggregated.
+
+## Aggregation
+An Aggregation describes how data collected is aggregated.
+
+The library MUST provide support for multiple types of Aggregations:
+* `Count`: indicates that data collected and aggregated with this `Aggregation` type will be turned
+into a count value.
+* `Sum`: indicates that data collected and aggregated with this `Aggregation` will be summed up.
+* `Mean`: indicates that data collected and aggregated with this `Aggregation` will calculate the
+mean value.
+* `Distribution`: indicates that the desired `Aggregation` is a histogram distribution. A
+distribution `Aggregation` may contain a histogram of the values in the population. User MUST
+define the bucket boundaries for that histogram.
+
+## View
+A View describes how individual measurements are both broken down (by a set of tag keys, which
+form the basis for a Metric labels.) and aggregated (e.g. into Distributions). A single Measure
+can be used by multiple Views : this allows for different aggregations and uses of a single base
+type.
+
+A View is defined from the following:
+* `name`: a string by which the View will be referred to, e.g. "rpc_latency". Names MUST be unique
+within the library.
+* `description`: a free format descriptive string, e.g. "RPC counts for last minute and hour"
+* `Measure`: the Measure to which this view is applied. This field MAY be replaced with
+`MeasureDescription` if defined.
```

Comment:
I'd not mention MeasureDescription, it is more of an implementation detail than the general rule.

Reply:
done.